src/actions/activities.js

```diff
@@ -0,0 +1,32 @@
+import uuid from 'uuid';
```

Comment:
probably you should check `uuid` docs. This version is depracated, if I remember correctly we should import from `uuid/v4`. But confirm taht

Reply:
yup, uuid-v4 installed